src/platform-apps/channels/message.tsx

```diff
@@ -206,6 +208,10 @@ export class Message extends React.Component<Properties, State> {
     }
   }
 
+  searchMentionableUsers = async (search: string) => {
+    return await searchMentionableUsersForChannel(this.props.channelId, search, this.props.users);
```

Comment:
I see that we are calling the api directly without using redux, any reason for that ?

all the api calls are in sagas

Reply:
This data seems like it's completely isolated to the individual component and user action. I didn't think there was any reason to bring the complexity of global state into the picture.